main.py

```diff
@@ -117,18 +38,27 @@ def main() -> int:
         params=model.parameters()
     )
 
+    experiment = Experiment(
+        api_key=os.environ['COMET_ML_API_KEY'],
+        project_name="deep-rl-pong",
+        workspace="thefebrin",
```

Comment:
nie podoba mi sie to hardcodowanie workspace :/ to jest i tak chyba twoje domyślne, więc nie wiem czy nie można tego pominąć